packages/hostScriptsServices/hostScripts/sensors.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
`sensors.sh` is not a good name either, it needs to be more self-explanatory. What does the script do?